<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MrqSensorPage</class>
 <widget class="QWidget" name="MrqSensorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="MrqUart" name="widget" native="true"/>
   </item>
   <item>
    <widget class="QCheckBox" name="chkAllEn">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>All SEN</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="MrqSubUart" name="tab">
      <attribute name="title">
       <string>SEN1</string>
      </attribute>
     </widget>
     <widget class="MrqSubUart" name="tab_2">
      <attribute name="title">
       <string>SEN2</string>
      </attribute>
     </widget>
     <widget class="MrqSubUart" name="tab_3">
      <attribute name="title">
       <string>SEN3</string>
      </attribute>
     </widget>
     <widget class="MrqSubUart" name="tab_4">
      <attribute name="title">
       <string>SEN4</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MrqUart</class>
   <extends>QWidget</extends>
   <header>mrquart.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MrqSubUart</class>
   <extends>QWidget</extends>
   <header>mrqsubuart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
